cli/cmd/start_cmd.go

```diff
@@ -49,6 +52,7 @@ func init() {
 	startCmd.Flags().StringVarP(&saveParams.organizationID, "organization", "", "", "organization id")
 	startCmd.Flags().StringVarP(&saveParams.environmentID, "environment", "", "", "environment id")
 	startCmd.Flags().StringVarP(&saveParams.agentApiKey, "api-key", "", "", "agent api key")
+	startCmd.Flags().StringVarP(&saveParams.tokenApiKey, "token", "", defaultToken, "token api key")
```

Comment:
`cli-api-key`instead of `token`?

Reply:
I'm kind of late to this discussion. But I think that `cli-api-key` is too close to `api-key` and it's hard to understand what's the difference between them. 

I would suggest using either `token` or a more meaningful name for this flag (which I have no suggestions yet). The idea behind using `token` is that the user will get the value from the `Token` tab, so will be more intuitive for the user to create a connection between the value and the flag they need to use.

Reply:
I agree with this statement too, I think we can go with token as it matches the SaaS cloud, then in the future we can decide to change it